packages/frontend-2/lib/workspaces/composables/management.ts

```diff
@@ -417,6 +423,76 @@ export function useCreateWorkspace() {
   }
 }
 
+export const useWorkspaceSso = (params: { workspaceSlug: string }) => {
+  const { result, loading, error } = useQuery<WorkspaceSsoCheckQuery>(
+    workspaceSsoCheckQuery,
+    { slug: params.workspaceSlug }
+  )
+
+  const hasSsoEnabled = computed(() => !!result.value?.workspaceBySlug.sso?.provider)
+  const provider = computed(() => result.value?.workspaceBySlug.sso?.provider ?? null)
```

Comment:
Does it need to explicitly `null`? `result.value?.workspaceBySlug.sso?.provider` will be `undefined` if the chaining fails, so you can also use it to check for falsey values